packages/playwright-core/src/server/dispatchers/jsHandleDispatcher.ts

```diff
@@ -16,15 +16,19 @@
 
 import type * as js from '../javascript';
 import type * as channels from '../../protocol/channels';
-import type { DispatcherScope } from './dispatcher';
 import { Dispatcher } from './dispatcher';
 import { ElementHandleDispatcher } from './elementHandlerDispatcher';
 import { parseSerializedValue, serializeValue } from '../../protocol/serializers';
+import type { PageDispatcher, WorkerDispatcher } from './pageDispatcher';
+import type { ElectronApplicationDispatcher } from './electronDispatcher';
+import type { BrowserContextDispatcher } from './browserContextDispatcher';
 
-export class JSHandleDispatcher extends Dispatcher<js.JSHandle, channels.JSHandleChannel> implements channels.JSHandleChannel {
+export type JSHandleDispatcherParentScope = BrowserContextDispatcher | PageDispatcher | WorkerDispatcher | ElectronApplicationDispatcher;
```

Comment:
How `BrowserContextDispatcher` could be a parent of a handle?

Reply:
That would be a BindingCallDispatcher's handle. Let me try rerouting it to the page. It'll be a semantics change though.